README.md

```diff
@@ -83,7 +83,41 @@ OK
 
 See `roachprod help <command>` for further details.
 
+# Setting up a cluster using roachprod/roachperf
+
+```bash
+# Create cluster with 4 nodes and local SSD (last node is used as load generator by roachperf)
+$ roachprod create test -n 4 --local-ssd
+Creating cluster marc-test with 4 nodes
+OK.
+  marc-test-0001	marc-test-0001.us-east1-b.cockroach-ephemeral	10.142.0.8	35.196.94.196
+  marc-test-0002	marc-test-0002.us-east1-b.cockroach-ephemeral	10.142.0.5	35.196.95.207
+  marc-test-0003	marc-test-0003.us-east1-b.cockroach-ephemeral	10.142.0.7	35.196.151.250
+  marc-test-0004	marc-test-0004.us-east1-b.cockroach-ephemeral	10.142.0.6	35.196.194.230
+Syncing...
+
+# Add gcloud SSH key
+$ ssh-add ~/.ssh/google_compute_engine
+
+# Stage scripts and binaries
+$ crl-stage-binaries marc-test all scripts
```

Comment:
It would be cool if we could roll these steps into `roachprod`. What's the point of a `roachprod` cluster that doesn't have a workable cockroach/loadgen setup on it?

Also, it would be useful to include the instructions to stage a local cockroach binary (`roachperf marc-test put cockroach`)

Reply:
moved to #6 and added the `put cockroach` example.
I agree that three tools (roachprod/crl-prod/roachperf) is getting unwieldy. My plan is to collapse it into at most 2, but that's for alter.